packages/dev/sharedUiComponents/src/nodeGraphSystem/graphNode.ts

```diff
@@ -454,6 +456,20 @@ export class GraphNode {
 
         outputs.push(...rightNode.content.inputs.filter((i) => !i.isConnected));
 
+        // Prioritize the already connected ports
+        const leftPortIndex = inputs.indexOf(leftPort);
+        const rightPortIndex = outputs.indexOf(rightPort);
+
+        if (leftPortIndex > 0) {
+            inputs.splice(leftPortIndex, 1);
+            inputs.splice(0, 0, leftPort);

```

Comment:
Just wondering if unshift would be better ?

Reply:
no idea lol

I still need to remove it anyway so it would be for the 2nd splice right?

Reply:
Well :)
![image](https://github.com/BabylonJS/Babylon.js/assets/1306056/de401652-8969-4bee-a1e7-eb67bf616b13)